.github/workflows/test-all.yml

```diff
@@ -0,0 +1,199 @@
+name: test-all
+
+on:
+    workflow_dispatch:
+    pull_request:
+    push:
+      tags:
+        - 'v*'
+    schedule:
+      - cron: '15 2 * * *'
```

Comment:
Can we add a note on why this is happening at 2:15AM every day (I suppose is the nightly)

Reply:
@cortinico Yes, this is the nightly. I just chose this time arbitrarily.